scripts/debian/builder-helpers.sh

```diff
@@ -186,254 +201,336 @@ copy_common_daemon_configs() {
 
   echo "------------------------------------------------------------"
   echo "--- Building mainnet legacy deb without keys:"
 
-  create_control_file mina-mainnet-legacy "${SHARED_DEPS}${DAEMON_DEPS}" 'Mina Protocol Client and Daemon' "${SUGGESTED_DEPS}"
+  create_control_file mina-mainnet-legacy "${SHARED_DEPS}${DAEMON_DEPS}" \
+    'Mina Protocol Client and Daemon' "${SUGGESTED_DEPS}"
 
   # Copy legacy binary
-  cp ./default/src/app/cli/src/mina_mainnet_signatures.exe "${BUILDDIR}/usr/local/bin/mina-legacy"
+  cp ./default/src/app/cli/src/mina_mainnet_signatures.exe \
+    "${BUILDDIR}/usr/local/bin/mina-legacy"
 
   build_deb mina-mainnet-legacy
 }
-##################################### END MAINNET LEGACY PACKAGE #######################################
+## END MAINNET LEGACY PACKAGE ##
 
-##################################### DEVNET LEGACY PACKAGE ############################################
+## DEVNET LEGACY PACKAGE ##
 build_daemon_devnet_legacy_deb() {
 
   echo "------------------------------------------------------------"
   echo "--- Building testnet signatures legacy deb without keys:"
 
-  create_control_file mina-devnet-legacy "${SHARED_DEPS}${DAEMON_DEPS}" 'Mina Protocol Client and Daemon for the Devnet Network' "${SUGGESTED_DEPS}"
+  create_control_file mina-devnet-legacy "${SHARED_DEPS}${DAEMON_DEPS}" \
+    'Mina Protocol Client and Daemon for the Devnet Network' "${SUGGESTED_DEPS}"
 
   # Copy legacy binary
-  cp ./default/src/app/cli/src/mina_testnet_signatures.exe "${BUILDDIR}/usr/local/bin/mina-legacy"
+  cp ./default/src/app/cli/src/mina_testnet_signatures.exe \
+    "${BUILDDIR}/usr/local/bin/mina-legacy"
 
   build_deb mina-devnet-legacy
 }
-##################################### END DEVNET LEGACY PACKAGE ########################################
+## END DEVNET LEGACY PACKAGE ##
 
-##################################### BERKELEY PACKAGE ##########################################
+## BERKELEY PACKAGE ##
 build_daemon_berkeley_deb() {
 
   echo "------------------------------------------------------------"
   echo "--- Building Mina Berkeley testnet signatures deb without keys:"
 
-  create_control_file "${MINA_DEB_NAME}" "${SHARED_DEPS}${DAEMON_DEPS}" 'Mina Protocol Client and Daemon for the Berkeley Network' "${SUGGESTED_DEPS}"
+  create_control_file "${MINA_DEB_NAME}" "${SHARED_DEPS}${DAEMON_DEPS}" \
+    'Mina Protocol Client and Daemon for the Berkeley Network' \
+    "${SUGGESTED_DEPS}"
 
   copy_common_daemon_configs berkeley testnet 'seed-lists/berkeley_seeds.txt'
 
   build_deb "${MINA_DEB_NAME}"
 
 }
-##################################### END BERKELEY PACKAGE ######################################
+## END BERKELEY PACKAGE ##
+
+copy_common_archive_configs() {
+  local ARCHIVE_DEB="${1}"
 
-##################################### ARCHIVE PACKAGE ###########################################
-build_archive_deb () {
-  ARCHIVE_DEB=mina-archive${DEB_SUFFIX}
+  cp ./default/src/app/archive/archive.exe \
```

Comment:
Comment: using the same binary name in each Debian package means that we can only have one on each machine, otherwise it would be overwritten.
I guess it is the expected behavior.